packages/lesswrong/components/shortform/NewShortformDialog.tsx

```diff
@@ -29,16 +29,16 @@ const NewShortformDialog = ({onClose, classes}: {
       open={open}
       onClose={onClose}
       fullWidth
-      maxWidth={isEAForum ? "md" : "sm"}
-      disableBackdropClick={isEAForum}
-      disableEscapeKeyDown={isEAForum}
+      maxWidth={isFriendlyUI ? "md" : "sm"}
+      disableBackdropClick={isFriendlyUI}
+      disableEscapeKeyDown={isFriendlyUI}
```

Comment:
What do these do?

Reply:
I believe the answer is to prevent someone from losing a bunch of their writing because they accidentally clicked outside of the modal.